framework/db/Command.php

```diff
@@ -537,7 +537,7 @@ public function createTable($table, $columns, $options = null)
     {
         $sql = $this->db->getQueryBuilder()->createTable($table, $columns, $options);
 
-        return $this->setSql($sql);
+        return $this->setSql($sql)->requireTableSchemaRefresh($table);
```

Comment:
@samdark Also this PR comes with a couple new test assertions that will demonstrate the issue. If this change is removed, this will fail:

https://github.com/yiisoft/yii2/pull/14073/files#diff-e48002914d2773120698a96387ecabd2R822